### PR TITLE
Add MIME type for RSS feeds

### LINKED ISSUE #60: Atom and RSS MIMEs are missing
While RSS is not registered in IANA it is pretty common format used by many sources. On the other hand Atom is fully speced and registered in IANA registry and is also used by some sources as a feed format.

### PR DESCRIPTION
**Pull request by hauleth**
Close #60

### CODE REVIEW

File: lib/mime.ex [hunk 1]
```diff
@@ -47,6 +48,7 @@ defmodule MIME do
     "application/vnd.openxmlformats-officedocument.wordprocessingml.document" => ["docx"],
     "application/vnd.rar" => ["rar"],
     "application/vnd.visio" => ["vsd"],
+    "application/wasm" => ["wasm"],
```
Comment (josevalim) [inline lib/mime.ex:51]: Fun tidybit: we don't have `w` in portuguese, it is an "imported" letter, so in the portuguese alphabet, it comes after `x`. Hence my mistake! :D

Does the same happen in polish with `v`?

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

File: lib/mime.ex [hunk 2]
```diff
@@ -94,14 +95,14 @@ defmodule MIME do
     "text/xml" => ["xml"],
     "video/3gpp" => ["3gp"],
     "video/3gpp2" => ["3g2"],
-    "video/quicktime" => ["mov"],
     "video/mp2t" => ["ts"],
     "video/mp4" => ["mp4"],
     "video/mpeg" => ["mpeg", "mpg"],
     "video/ogg" => ["ogv"],
+    "video/quicktime" => ["mov"],
```
Comment (josevalim) [inline lib/mime.ex:102]: I have no excuses for this one though! haha

File: lib/mime.ex (hunk 1 above)
Comment (hauleth) [reply lib/mime.ex:51]: Nope. Fortunately when someone encountered such problem they went with most reasonable solution - just use Latin order and add accented letters after their visual counterparts (so `ó` is after `o` not `u`). So `v` is between `u` and `w` and `x` is between `w` and `z`.

**Pull request merged**